bash_completion

```diff
@@ -3207,6 +3207,22 @@ _comp_xfunc()
     "$xfunc_name" "${@:3}"
 }
 
+# Call a POSIX-compatible awk.  Solaris awk is not POSIX-compliant, but Solaris
+# provides a POSIX-compatible version through /usr/xpg4/bin/awk.  We switch the
+# implementation to /usr/xpg4/bin/awk in Solaris if any.
+# @since 2.12
+if [[ ${OSTYPE-} == solaris* && -x /usr/xpg4/bin/awk ]]; then
+    _comp_awk()
+    {
+        /usr/xpg4/bin/awk "$@"
+    }
+else
+    _comp_awk()
+    {
+        awk "$@"
```

Comment:
Now that we're doing this, maybe...?
```suggestion
        command awk "$@"
```

Reply:
There are many occurrences of `awk` in the codebase. Can I apply the change to all the occurrences in a separate PR? Or do we use `command` only in this case?

Reply:
Or another way is to replace all the other occurrences of `awk` with `_comp_awk`.

Reply:
Got it.

Reply:
e18667a9a17d7db0b480426311ad0fe911c26dc3..302dc52b9b02dc9a8fa9b1833012b3c4f96b6d7b